index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <title>Explore the Tour: Warner Bros Studio Tour</title>
+</head>
+<body>
+  <header id="landing">
+    <nav>
+      <div class="logo">
```

Comment:
- [ ] Kindly add this missing `social-icons`  section to your navigation bar to match the original design

![image](https://user-images.githubusercontent.com/98453979/204053070-e57f0dd0-894e-42d1-bd6e-619464944555.png)

Reply:
Kindly add this `social-icons` section to your navigation bar on the about page also. Currently, this section is only added to the home page. But as per the design provided by the microverse, this section is part of all the pages. Kindly do so and then submit for a re-review.